### PR TITLE
Remove PG11

### PR DESCRIPTION
**Pull request by hughcapet**
Remove the EOL PG11 and decrease the complexity a bit

### CODE REVIEW

File: postgres-appliance/tests/test_spilo.sh [hunk 1]
```diff
@@ -359,8 +342,8 @@ function test_spilo() {
 
     # TEST SUITE 6
     local basebackup_container
-    basebackup_container=$(start_clone_with_basebackup_upgrade_container "$upgrade_container")  # SCOPE=upgrade2 PGVERSION=13 CLONE: _SCOPE=upgrade
-    log_info "[TS6] Started $basebackup_container for testing major upgrade 12->13 after clone with basebackup"
+    basebackup_container=$(start_clone_with_basebackup_upgrade_container "$upgrade_container")  # SCOPE=upgrade2 PGVERSION=14 CLONE: _SCOPE=upgrade
+    log_info "[TS6] Started $basebackup_container for testing major upgrade 12->14 after clone with basebackup"
```
Comment (idanovinda) [inline postgres-appliance/tests/test_spilo.sh:346]: 13->14 ?

Comment (hughcapet) [inline postgres-appliance/tests/test_spilo.sh:346]:
```suggestion
    log_info "[TS6] Started $basebackup_container for testing major upgrade 13->14 after clone with basebackup"
```

**Review by idanovinda — APPROVED** [review]

**Comment by idanovinda** [comment]
👍

**Comment by hughcapet** [comment]
👍

**Pull request merged**